app/presenters/nav_tab/link_collection.rb

```diff
@@ -5,13 +5,14 @@ class NavTab::LinkCollection
   include Rails.application.routes.url_helpers
   include TranslationHelper
 
-  attr_reader :ability, :facility
+  attr_reader :ability, :facility, :user
 
   delegate :single_facility?, to: :facility
 
-  def initialize(facility, ability)
+  def initialize(facility, ability, user)
```

Comment:
Would it be worth making these keyword args?

(I won't lie, it took me a few minutes to remember the term "keyword argument". This is what 6 months away will do 😜)